### PR TITLE
rebase: update kubernetes to v1.25.0

### PR DESCRIPTION
**Pull request by Madhu-1**
update kubernetes to the latest [v1.25.0 release](https://github.com/kubernetes/kubernetes/releases/tag/v1.25.0).

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Review by yati1998 — DISMISSED** [review]

**Comment by Madhu-1** [comment]
> /retest ci/centos/k8s-e2e-external-storage/1.22

Rook deployment failure

File: go.mod [hunk 1]
```diff
@@ -21,14 +21,14 @@ require (
...
-	github.com/prometheus/client_golang v1.12.2
+	github.com/onsi/ginkgo/v2 v2.1.4
+	github.com/onsi/gomega v1.19.0
+	github.com/prometheus/client_golang v1.12.1
 	github.com/stretchr/testify v1.8.0
 	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
 	golang.org/x/net v0.0.0-20220722155237-a158d28d115b
 	golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f
-	google.golang.org/grpc v1.48.0
+	google.golang.org/grpc v1.47.0
```
Comment (Rakshith-R) [inline go.mod]: Any reason why gomega, client_golang & grpc packages were downgraded ? 
were these done automatically ?

Comment (Madhu-1) [reply go.mod]: Good catch. it was due to rebase, fixed it now

**Review by Rakshith-R — DISMISSED** [review]

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Comment by Madhu-1** [comment]
CI passed :tada:

**Review by Rakshith-R — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Pull request merged**